src/client/article.ts

```diff
@@ -123,7 +125,15 @@ function insertEpic(): void {
     }
 }
 
-
+function footerInit(): void {
+    const isAndroid = /(android)/i.test(navigator.userAgent);
+    const footer = document.getElementById('articleFooter')
+    if (footer && isAndroid){
+        ReactDOM.unmountComponentAtNode(footer)
+    } else {
+        isCCPA();
```

Comment:
Should this be called something like `renderFooter`? `isCCPA` sounds like it should return a boolean.

Reply:
isCCPA was part of the previous footer commit, so we can change this once updating the footer design